src/core/datatypes/phylogenetics/DistanceMatrix.cpp

```diff
@@ -88,7 +88,7 @@ size_t DistanceMatrix::getSize(void) const
 }
 
 
-std::string DistanceMatrix::getFilename(void) const
+path DistanceMatrix::getFilename(void) const
```

Comment:
Yeah, it can.  I didn't change it, because it was not originall `const std::string&`.  Should I switch this to `const path&`, and also the ones below?

Reply:
I'm changing it.